osgar/tools/lidarview.py

```diff
@@ -182,7 +182,7 @@ def get_image(data):
             img = np.array(depth2danger(data) * 255, dtype=np.uint8)
             im_color = cv2.cvtColor(img, cv2.COLOR_GRAY2RGB)
         else:
-            img = np.array(data/40, dtype=np.uint8)
+            img = np.array(np.minimum(255*40, data)/40, dtype=np.uint8)
```

Comment:
I would use `min(255, data/40)`??

Reply:
How would that work? Is that python min? Taking minimum from a number and array... I don't know. What I have, works.

Reply:
you are right, I forgot it is numpy array ...